server/src/iec62209_service/routers/analysis_creation.py

```diff
@@ -198,6 +199,9 @@ async def analysis_creation_pdf(tmp=Depends(texutils.create_temp_folder)) -> Res
                 )
             )
 
+        with open(texpath / "version.tex", "w") as fout:
```

Comment:
TIP: `` (texpath / "version.txt").write_text(info.__version__) ``

Reply:
I prefer having `*.in` files so that the files tracked by git don't get modified when the version changes, but I am not opposed to doing it in-place, which I can easily do adapting the `sed` calls.

If it's OK for you, I'll keep the `update_version.sh` script but do everything in-place. I do not like to depend on external packages for something so trivial as a string search-and-replace.

Reply:
this is your service  ... so you have the last word :-)